apps/condo/domains/property/schema/Property.js

```diff
@@ -283,7 +282,7 @@ const Property = new GQLListSchema('Property', {
             },
         },
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
+    plugins: [uuided(), addressService('address', { address: addressFieldHooks }), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
```

Comment:
why we need to owerride the field name? it looks like we use `address` everywhere

Reply:
I thought about `Ticket.propertyAddress` and `Ticket.propertyAddressMeta`.
But now I think this overriding may not need. If yes, I'll remove the code for overriding.

Reply:
propertyAddress and propertyAddressMeta are not the same as addressService plugin fields. It's just cache fields with their own logic. You can't set it directly. And you will not use your plugin for that fields. It's better to remove this redundant extensibility.